src/setup/node.rs

```diff
@@ -274,21 +274,45 @@ pub struct Node {
 impl Node {
     pub fn builder() -> NodeBuilder {
         NodeBuilder::stateful()
-            .map_err(|e| format!("Unable to create builder: {e:?}"))
+            .map_err(|e| format!("unable to create builder: {e:?}"))
             .unwrap()
     }
 
     pub fn stop(&mut self) -> io::Result<ChildExitCode> {
         match self.child.try_wait()? {
             None => self.child.kill()?,
-            Some(code) => return Ok(ChildExitCode::ErrorCode(code.code())),
+            Some(status) => return Ok(ChildExitCode::ErrorCode(status.code())),
         }
```

Comment:
info for reviewers:

renamed `code` to `status` because the [try_wait](https://doc.rust-lang.org/std/process/struct.Child.html#method.try_wait) returns:
```
If the child has exited, then Ok(Some(status)) is returned. If the exit status is not available at this time then Ok(None) is returned. If an error occurs, then that error is returned.
```

also, `exit` renamed to `exit_status` because wait() function returns `ExitStatus`